<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsForm</class>
 <widget class="QWidget" name="SettingsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;
font-family: &quot;Microsoft YaHei&quot;;
color: #535353;
QPushButton
{
border:none;
}</string>
  </property>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>屏幕截图保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>51</x>
     <y>280</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>视频录像保存</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>视频设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>90</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>音频设置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>170</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-weight:bold;</string>
   </property>
   <property name="text">
    <string>控制设置</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>337</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QCheckBox" name="videoChk">
      <property name="text">
       <string>视频同步</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>视频编码:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="videoCodecList"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>视频来源:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="videoSourceList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>337</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QCheckBox" name="audioChk">
      <property name="text">
       <string>音频同步</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>音频编码:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="audioCodecList"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>音频来源:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="audioSourceList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>341</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QCheckBox" name="controlChk">
      <property name="text">
       <string>允许控制</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="stayAwakeChk">
      <property name="text">
       <string>保持唤醒</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="clipBoardChk">
      <property name="text">
       <string>剪切板同步</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>140</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>音频采样率:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="audioBitRateList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>60</y>
     <width>281</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>视频采样率:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="videoBitRateList"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>视频帧速:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="frameRateList"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>341</width>
     <height>38</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="styleSheet">
       <string notr="true">border: none;
background-color: #2965fa;
border-radius: 3px;
color: #ffffff;
font-family: &quot;Microsoft YaHei&quot;;
font-weight: bold;
padding: 3px 10px 3px 10px;</string>
      </property>
      <property name="text">
       <string>更改目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_3">
      <property name="styleSheet">
       <string notr="true">border: none;
color: #2965fa;
font-family: &quot;Microsoft YaHei&quot;;
font-weight: bold;
background:none;
padding: 5px 10px
</string>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>300</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <widget class="QLineEdit" name="lineEdit_2"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_2">
      <property name="styleSheet">
       <string notr="true">border: none;
background-color: #2965fa;
border-radius: 3px;
color: #ffffff;
font-family: &quot;Microsoft YaHei&quot;;
font-weight: bold;
padding: 3px 10px 3px 10px;</string>
      </property>
      <property name="text">
       <string>更改目录</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_4">
      <property name="styleSheet">
       <string notr="true">border: none;
color: #2965fa;
font-family: &quot;Microsoft YaHei&quot;;
font-weight: bold;
background:none;
padding: 5px 10px
</string>
      </property>
      <property name="text">
       <string>打开</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
